target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -150,12 +154,16 @@ case "$FIRMWARE" in
 	openmesh,a62 |\
 	qxwlan,e2600ac-c1 |\
 	qxwlan,e2600ac-c2)
-		ath10kcal_extract "0:ART" 4096 12064
```

Comment:
Please don't change/break other boards (since this would also require a DTS update in qcom-ipq4019-a62.dts). But this is outside the scope of this patch.